src/PointwiseFunctions/AnalyticSolutions/Poisson/Solutions.hpp

```diff
@@ -0,0 +1,16 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines Poisson::Solutions
+
+#pragma once
+
+namespace Poisson {
+/*!
+ * \ingroup AnalyticSolutionsGroup
```

Comment:
We don't have this group :)

Reply:
It's added in #879, fixed that according to your suggestions

Reply:
We may not be adding it. Please remove it unless you want this to block on a consensus about adding it. (Similar to the Poisson equation itself.{

Reply:
done